Sources/tart/OCI/Manifest.swift

```diff
@@ -5,6 +5,7 @@ let ociConfigMediaType = "application/vnd.oci.image.config.v1+json"
 
 // Annotations
 let uncompressedDiskSizeAnnotation = "org.cirruslabs.tart.uncompressed-disk-size"
+let uploadTimeAnnotation = "org.cirruslabs.tart.upload-time"
```

Comment:
Looks more consistent with the rest of the code. What do you think?

Will also need to rename the `uploadTimeAnnotation` accordingly.

```suggestion
let uploadTimeAnnotation = "org.cirruslabs.tart.upload-date"
```

Reply:
Not sure to who is the question directed, but my 2 cents on my reasoning:
- Docker (and OCI spec) have `created` attribute. I considered `org.cirruslabs.tart.uploaded`, but it didn't seem right (I felt as if it could also be a boolean)
- I considered `….uploaded-at` (which would be the clearest to me), but it felt like breaking out of all conventions
- between `upload-time` and `upload-date` I picked `upload-time`, because I thought it hints at the fact it's not just a date (and the `Date` suffix in Swift variables are, to me, borrowed from `Date` class name, not name of the datetime concept per se).

I don't have strong opinion either way!